packages/ui/src/utils/get-user-updated-properties-schema.ts

```diff
@@ -0,0 +1,41 @@
+import { isDefined } from './is-defined';
+
+export function getUserUpdatedPropertiesSchema(
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  obj1: Record<string, any>,
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  obj2: Record<string, any>,
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+): Record<string, any> {
```

Comment:
These parameter names are not very self-explanatory. Would it be possible to use more descriptive names?

As an example, `ob1` seems to represent *schema properties*, could this parameter be renamed to `schemaProperties` instead?

For `obj2`, it looks like is the model created form the user, we can reuse the name you're using in the test, `inputModel`

Then the function signature could read as:

```typescript
export function getUserUpdatedPropertiesSchema(
  // eslint-disable-next-line @typescript-eslint/no-explicit-any
  schemaProperties: Record<string, any>,
  // eslint-disable-next-line @typescript-eslint/no-explicit-any
  inputModel: Record<string, any>,
  // eslint-disable-next-line @typescript-eslint/no-explicit-any
): Record<string, any> {
```